### DISCUSSION TITLE
Colour entries of menus other than index

### DISCUSSION THREAD

**Discussion by whitney-cumber** [neomutt/neomutt#4526] [Development]
## What would you like to be able to do?

Colour the entries of the pgp/smime key selection menu, e.g. to mark untrusted keys, expired keys, or my own keys.
By extension it should also be possible to colour other menus, e.g. file browser (files vs directories), mailbox browser (new mail), attachments menu, ...

## What needs to change in NeoMutt?

I imagine a similar mechanism to how index works, which allows both colour individual columns, e.g. the trust column as well as the whole line (e.g. fade out expired keys).  However, to me the current approach taken there (allowing to name some expandos) seems not scalable well.  It also does not allow to colour different results of flags, e.g. `%t` expands to `f` for a trusted key (should be coloured green) and `n` for an untrusted key (should be red).

## Additional Info

Similar to:

https://github.com/neomutt/neomutt/issues/3686 (individual status line colouring)
https://github.com/neomutt/neomutt/issues/3680 (colouring headers)
https://github.com/neomutt/neomutt/issues/3672 (status_format caveats)
https://github.com/neomutt/neomutt/issues/3660 (allow different colours in attachment menu)
https://github.com/neomutt/neomutt/issues/3617 (allow different colours in sidebar)

**Comment by flatcap** [comment]
I'm reminded of a Vim internet meme from long ago:

![vim-syntax](https://pbs.twimg.com/media/BwLLk8PCQAIfleK?format=jpg)
